AirBench/Views/Bench/Index.cshtml

```diff
@@ -35,36 +41,43 @@
         <th></th>
     </tr>
 
-@foreach (var item in Model) {
-    <tr>
+@*@foreach (var item in Model) {
+    <tr class="inner-list">
         <td>
             @Html.DisplayFor(modelItem => item.Description)
         </td>
-        <td>
+        <td id="Number">
             @Html.DisplayFor(modelItem => item.NumberOfSeats)
         </td>
         <td>
             @Html.DisplayFor(modelItem => item.CreatorUserId)
         </td>
+        <td>
+            @Html.DisplayFor(modelItem => item.Name)
+        </td>
         <td>
             @Html.DisplayFor(modelItem => item.Latitude)
         </td>
         <td>
             @Html.DisplayFor(modelItem => item.Longitude)
         </td>
         <td>
-            @Html.DisplayFor(modelItem => item.Rating)
+            @if ((item.Rating > (decimal) 0.0)){
```

Comment:
Instead of an explicit cast, you can use `m` to denote a `decimal` literal (i.e. `0m`).